### PR TITLE
*: use coreos/go-json

### PR DESCRIPTION
**Pull request by darkmuggle**
Use `coreos/go-json` instead of `ajeddeloh/go-json`.  

Before Andrew left, he created `coreos/go-json` from his repo. We should be using the CoreOS sources.

### CODE REVIEW

**Comment by darkmuggle** [comment]
After this merges, I'll update `fcct` which is why I was playing in this code.

**Review by bgilbert — APPROVED** [review]

**Comment by bgilbert** [comment]
@darkmuggle Yup.  You should have merge access; LMK if not.

**Review by ashcrow — APPROVED** [review]

**Pull request merged**
